### DISCUSSION TITLE
Archievements<title>

### DISCUSSION THREAD

**Discussion by CodesdaLu** [community/community#43250] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi, 
how to get the achievements ?

**Comment by LuisContrerasGlz** [comment, marked as answer]
Hi,

Here is the link for how to get them: https://github.com/Schweinepriester/github-profile-achievements

I am actually trying to get the galaxy brain which you get by getting at least 2 people to mark your answer as the answer, with the arrow button, I am trying to get that one right now :-) Hope this helps

**Comment by LuisContrerasGlz** [comment]
This is the one I am talking about: https://github.com/community/community/discussions/28656

**Reply by CodesdaLu** [reply]
I'm also trying this. rs

**Reply by LuisContrerasGlz** [reply]
Awesome, wish you all the best ! you got it! :-D

**Comment by github-staff** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
